frontend/package.json

```diff
@@ -12,10 +12,11 @@
     "dagre": "^0.8.2",
     "http-proxy-middleware": "^0.19.0",
     "immer": "^1.7.4",
+    "jss": "^10.0.0-alpha.3",
```

Comment:
Are we using jss?

Reply:
No, this is a workaround for https://github.com/mui-org/material-ui/issues/14040. Once that issue is fixed, we should be able to remove this direct dep

Reply:
Ah interesting.
You can consider using an older version (e.g. 3.7.x) that moves us very close to being updated without needing this hack. Keep in mind that even with the latest, we'll be outdated again in just a week or two. Up to you tho.

Reply:
I set it to 3.7.1 so we don't need the workaround, but we'll need to remember to update it in the future